core/block_svg.js

```diff
@@ -617,9 +618,9 @@ Blockly.BlockSvg.prototype.snapToGrid = function() {
  * @return {!Blockly.utils.Rect} Object with coordinates of the bounding box.
  */
 Blockly.BlockSvg.prototype.getBoundingRectangle = function() {
-  var blockXY = this.getRelativeToSurfaceXY();
-  var blockBounds = this.getHeightWidth();
-  var left, right;
+  const blockXY = this.getRelativeToSurfaceXY();
+  const blockBounds = this.getHeightWidth();
+  let left, right;
```

Comment:
One declaration per line, as per [5.1.2](https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration) (I'm surprised clang doesn't catch this).

Reply:
Done